std/assembly/math.ts

```diff
@@ -2046,6 +2053,11 @@ export namespace NativeMathf {
     }
   }
 
+  @inline
+  export function signbit(x: f32): bool {
+    return <bool>((reinterpret<u32>(x) >>> 31) & <i32>(x == x));
```

Comment:
This looks like it might result in an unnecessary `& 1` to convert from i32 to bool. A better way to write it is: `<bool>(reinterpret<u32>(x) >>> 31) & (x == x)`, as it keeps type conversions to a minimum.